### ISSUE TITLE
Building 2016.04.05 from source tarball fails

### ISSUE THREAD

**Issue by ryandesign** [rg3/youtube-dl#9088]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.04.05*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.04.05**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

```
sudo port -d install youtube-dl
...
--->  Building youtube-dl
DEBUG: Executing proc-pre-org.macports.build-build-0
DEBUG: Executing org.macports.build (youtube-dl)
DEBUG: Environment: 
CC_PRINT_OPTIONS='YES'
CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/_Users_rschmidt_macports_dports_net_youtube-dl/youtube-dl/work/.CC_PRINT_OPTIONS'
CPATH='/opt/local/include'
LIBRARY_PATH='/opt/local/lib'
MACOSX_DEPLOYMENT_TARGET='10.11'
DEBUG: system:  cd "/opt/local/var/macports/build/_Users_rschmidt_macports_dports_net_youtube-dl/youtube-dl/work/youtube-dl" && /usr/bin/make -w all PYTHON=/opt/local/bin/python2.7 
make: Entering directory `/opt/local/var/macports/build/_Users_rschmidt_macports_dports_net_youtube-dl/youtube-dl/work/youtube-dl'
zip -q youtube-dl youtube_dl/*.py youtube_dl/*/*.py
zip -q -j youtube-dl youtube_dl/__main__.py
echo '#!/opt/local/bin/python2.7' > youtube-dl
cat youtube-dl.zip >> youtube-dl
rm youtube-dl.zip
chmod a+x youtube-dl
/opt/local/bin/python2.7 devscripts/make_contributing.py README.md CONTRIBUTING.md
/opt/local/bin/python2.7 devscripts/make_issue_template.py .github/ISSUE_TEMPLATE_tmpl.md .github/ISSUE_TEMPLATE.md
Traceback (most recent call last):
  File "devscripts/make_issue_template.py", line 29, in <module>
    main()
  File "devscripts/make_issue_template.py", line 16, in main
    with io.open(infile, encoding='utf-8') as inf:
IOError: [Errno 2] No such file or directory: '.github/ISSUE_TEMPLATE_tmpl.md'
make: *** [ISSUE_TEMPLATE.md] Error 1
```

---

### Description of your *issue*, suggested solution and other information

I am the maintainer of youtube-dl in the MacPorts package management system, but building youtube-dl 2016.04.05 from a source tarball fails. This problem did not exist in youtube-dl 2016.03.27.

**Comment by jaimeMF** [comment]
Should be fixed in the next release. Thanks for the report and sorry for the issue.

@phihag the `make` command failed with the last tar file, I think it's worth to make a new release for package managers that run it.

**Comment by phihag** [comment]
Fixed in 2016.04.06.
